src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -1060,19 +1060,41 @@ public static void AnalyzeTargetHelper(TContext context, IEnumerable<Skimmer<TCo
                         ? Stopwatch.StartNew()
                         : null;
 
+                    if (context.Traces.HasFlag(DefaultTraces.MemoryUsage))
```

Comment:
>context [](http://example.com/codeflow?start=24&length=7)

should we move this earlier in the method, before starting the stop watch? i.e., we don't need to include this code in the timing data.

Reply:
fixed